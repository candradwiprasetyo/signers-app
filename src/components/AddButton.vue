<template>
  <button
    @click="$emit('click')"
    class="px-4 py-2 bg-red-400 hover:bg-red-500 text-white rounded-lg shadow text-xs font-bold"
  >
    TAMBAH
  </button>
</template>
