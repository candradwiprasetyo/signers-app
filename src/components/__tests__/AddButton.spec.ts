import { describe, it, expect } from 'vitest'
import { mount } from '@vue/test-utils'
import AddButton from '@/components/AddButton.vue'

describe('AddButton.vue', () => {
  it('render correctly button', () => {
    const wrapper = mount(AddButton)
    expect(wrapper.html()).toMatchSnapshot()
  })
})
