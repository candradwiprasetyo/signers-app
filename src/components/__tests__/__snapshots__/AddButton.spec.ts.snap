// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`AddButton.vue > render correctly button 1`] = `"<button class="px-4 py-2 bg-red-400 hover:bg-red-500 text-white rounded-lg shadow text-xs font-bold"> TAMBAH </button>"`;
